ConfigVersionControl/git_version_control.py

```diff
@@ -153,6 +157,9 @@ def commit(self, commit_comment):
         Args:
             commit_comment (str): comment to leave with the commit
         """
+        if len(self.repo.index.diff("HEAD")) == 0:
+            print "GIT: Nothing to commit"
```

Comment:
As above, `print_and_log`

Reply:
Thanks - i'd put them in for diagnostics and wasn't sure of they would stay, but it seems that the blockserver passes the directory name rather than each file so its not too much additional information.